study.md

```diff
@@ -160,15 +170,21 @@ What is a response? What does it contain? What are some common status codes in a
 response and what do they mean?
 
 ```md
-<!-- your answer here -->
+<!-- data sent from server bundled with a line of data/header and body.
+
+header sometimes has additional info like host, what kind of formats allowed,
+application; body can be empty
+
+common status code:  200 - accepted; 404 - page not found-->
 ```
 
 ## Make a curl request
 
 Using curl, how would you get the content from Reddit.com?
 
 ```md
-<!-- your answer here -->
+<!-- curl http://reddit.com
+this looks awful; not sure I'm doing that correctly -->
```

Comment:
You were close here, the correct curl request would be:

`curl --request GET https://www.reddit.com`